setup/discovery_server.md

```diff
@@ -102,8 +104,54 @@ turtlebot4-setup
+
+<b>Setup instructions:</b>
+- Update to the [latest firmware](https://iroboteducation.github.io/create3_docs/releases/overview/) using the webserver.
+- Once updated, perform a [factory reset](https://iroboteducation.github.io/create3_docs/webserver/about/#:~:text=set%20to%20USB.-,Factory%20Reset,-A%20hyperlink%20to) to disconnect the Create® 3 from any Wi-Fi networks.
+
+```note
+Ensure that this factory reset is done before the remaining setup as it will clear out the settings that are written to the Create® 3 by the Raspberry Pi.
+```
+
+<u><b style="font-size: 20px;">Raspberry Pi</b></u>
+
+The Raspberry Pi needs to be configured to host a discovery server, and the TurtleBot 4 upstart job needs to be reinstalled with the new configuration. During this step, the Raspberry Pi will also write the discovery server details to the Create® 3.
+
+Unlike ROS 2 Humble, the Jazzy version always uses the `create3_republisher` node to broadcast the Create® 3 topics, services, and actions from the Raspberry Pi. For more information on the `create3_republisher` see the [Create® 3 Republisher section of the Create® 3 page](../software/create3.md#create-3-republisher).
+
+<b>Setup instructions:</b>
+- SSH into the Raspberry Pi
+- Run the [TurtleBot 4 setup tool](../software/turtlebot4_setup.md#configuration-tools):
+
+```bash
+turtlebot4-setup
+```
+- Enter the <b>Discovery Server</b> menu via <b>ROS Setup</b>.
+- Set Enabled to `True`.
+- Onboard Discovery Server: This is the primary server that will connect the onboard robot nodes.
+  - Leave onboard server port as `11811`.
+  - Set the onboard server ID as a unique number in the system. If there is only one robot then this can be left as `0`. If there is more than one robot then give each robot a unique number between 0 and 255.
+- Offboard Server: This allows support for the robots to communicate on an offboard server but is not recommended for beginners because it can overload the network when there are too many robots.
+  - Leave all of these as default. When the offboard IP address is blank then all offboard server settings will be ignored.
+- Save the settings, navigate to the main menu, and apply settings.
+- Exit the setup tool
+- In your terminal, call `turtlebot4-source` to apply the new settings to your terminal.
+- Restart the ROS 2 daemon with `turtlebot4-daemon-restart` or `ros2 daemon stop; ros2 daemon start`
+- When the Create® 3 chimes, call `ros2 topic list` to view topics
+
+```tip
+When a network change occurs or a robot ROS service or robot is rebooted, you may need to restart the ros2 daemon to see the changes with the ROS 2 command line tools (ros2cli).
+You may also need to call `ros2 topic list` twice to get a full list of topics. This is because the first time that you run the command, it starts the daemon which will record all of the available topics. If there are a lot of ROS nodes then it may take some time before the full topic list is available.
```

Comment:
Same as above: Superfluous new line or maybe this should be two paragraphs (flagged because the space was removed as part of that other discussion)